internal/storage/storageutil/client.go

```diff
@@ -28,18 +28,27 @@ import (
 )
 
 type StorageClientConfig struct {
-	ClientProtocol             mountpkg.ClientProtocol
+	/** Common client parameters. */
+
+	// ClientProtocol decides the go-sdk client to create. Based on the value,
+	// either http client or grpc client is created.
```

Comment:
nit: I think we can remove "Based on the value, either http client or grpc client is created."